app/js/eventteam.js

```diff
@@ -125,6 +128,12 @@ angular.module("teamform-eventteam-app", ["firebase", "ngMaterial"])
         return teams.filter(function(team) {return team.skillsMatch.number > 0;});
     };
 
+    // filter teams that missing the signed in user skills
+    $scope.filterMissingSkillsMatch = function(teams) {
+        console.log("filterMissingSkillsMatch()");
+        return teams.filter(function(team) {return team.missingSkillsMatch.number > 0;});
+    };
+
     // filter the teams
     $scope.filterTeams = function(filterPlacesSwitch, filterSkillsMatchSwitch) {
```

Comment:
change the signature to `function(filterPlacesSwitch, filterSkillsMatchSwitch, filterMissingSkillsMatchSwitch)`